webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -149,7 +149,7 @@
                             </tr>
                             <tr>
                                 <th scope="row">OS</th>
-                                <td>{{ report.os_name }}</td>
+                                <td>{{ report.os_pretty_version or report.os_name }}</td>
```

Comment:
Also, ``report`` cannot be undefined. And if it ever is, there will be plenty of errors on that page. We can't be over protective with everything, we sometimes have to trust that some value will be there. And since ``os_pretty_version`` is set by the processors, it should always be there. Otherwise we have a different problem to solve.